src/Content/Conversation.php

```diff
@@ -492,19 +488,19 @@ public function create(array $items, string $mode, $update, bool $preview = fals
 				}
 			}
 		} elseif ($mode === 'notes') {
-			$items = $this->addChildren($items, false, $order, $this->userSession->getLocalUserId(), $mode);
+			$items = $this->addChildren($items, false, $order, $this->session->getLocalUserId(), $mode);
 
 			if (!$update) {
 				$live_update_div = '<div id="live-notes"></div>' . "\r\n"
-					. "<script> var profile_uid = " . $this->userSession->getLocalUserId()
+					. "<script> var profile_uid = " . $this->session->getLocalUserId()
 					. "; var netargs = '/?f='; </script>\r\n";
 			}
 		} elseif ($mode === 'display') {
 			$items = $this->addChildren($items, false, $order, $uid, $mode);
 
 			if (!$update) {
 				$live_update_div = '<div id="live-display"></div>' . "\r\n"
-					. "<script> var profile_uid = " . $this->session->get('uid', 0) . ";"
+					. "<script> var profile_uid = " . $this->session->getLocalUserId() ?? 0 . ";"
```

Comment:
`getLocalUserId` never returns `null`. Also, removing ternary operator ambiguity with concatenation operator.
```suggestion
					. "<script> var profile_uid = " . ($this->session->getLocalUserId() ?: 0) . ";"
```